lib/utils/basic.d.ts

```
@@ -2,16 +2,29 @@ import type { ComponentPublicInstance } from 'vue';
     <T extends {}, U>(target: T, source: U): T & U;
-    <T_1 extends {}, U_1, V>(target: T_1, source1: U_1, source2: V): T_1 & U_1 & V;
-    <T_2 extends {}, U_2, V_1, W>(target: T_2, source1: U_2, source2: V_1, source3: W): T_2 & U_2 & V_1 & W;
+    <T extends {}, U, V>(target: T, source1: U, source2: V): T & U & V;
+    <T extends {}, U, V, W>(target: T, source1: U, source2: V, source3: W): T & U & V & W;
     (target: object, ...sources: any[]): any;
 };
 export declare const inBrowser: boolean;
-export declare type Numeric = number | string;
-export declare type ComponentInstance = ComponentPublicInstance<{}, any>;
+export type Numeric = number | string;
+export type ComponentInstance = ComponentPublicInstance<{}, any>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)